fdbclient/NativeAPI.actor.cpp

```diff
@@ -6869,6 +6872,9 @@ ACTOR Future<bool> checkSafeExclusions(Database cx, std::vector<AddressExclusion
 		throw;
 	}
 	TraceEvent("ExclusionSafetyCheckCoordinators").log();
+	if (cx->getConnectionRecord()->hasUnresolvedHostnames()) {
```

Comment:
I don't have any thumb of rule, so what I did was to check everywhere before we use it, because checking is just checking a boolean field, and it doesn't increase the total resolve times. It hurts readability though, I admit.